gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ adventure = "4.12.0"
 kotlin = "1.7.22"
 hydrazine = "1.7.2"
 dependencyGetter = "v1.0.1"
-minestomData = "1c1921cd41"
+minestomData = "53e0da5be1"
```

Comment:
Needs to move to [this](https://github.com/Minestom/MinestomDataGenerator/commit/ddde11056e4f6b6408b0a3d159abdab12c3e0633) commit. The one written here is for the 1.19.3 branch, so it will be deleted by Jitpack soon enough